jobs/config.json

```diff
@@ -8157,9 +8157,9 @@
       "--env-file=jobs/platform/kops_aws.env",
       "--env-file=jobs/env/ci-kubernetes-e2e-kops-aws.env",
       "--extract=ci/latest",
-      "--mode=docker",
```

Comment:
you'll need to move the config from https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml#L58-L71 to around https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L15079

Reply:
Oh, right. Der.